releasenotes/notes/flask-streaming-e6dad359a155fdb3.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    flask: Adds support for streamed responses by using a WSGI middleware. Note that additional WSGI spans, `flask.application` and `flask.response` will be generated.
```

```suggestion
    flask: Adds support for streamed responses. Note that additional WSGI spans, ``flask.application`` and ``flask.response`` will be generated.
```

Does it matter that it is using a WSGI middleware? that is just an internal implementation detail no?

---

We'll need the double back ticks though